### ISSUE TITLE
Top-level string in replicants should be described with `"`

### ISSUE THREAD

**Issue by Hoishin** [nodecg/nodecg#509]
### Steps to reproduce

1. Create a bundle with this extension
```js
module.exports = function (nodecg) {
	const fooRep = nodecg.Replicant('foo')
	console.log(fooRep.value)
	console.log(typeof fooRep.value)
};
```

2. Launch NodeCG
-> Logs `undefined`

3. Stop and re-launch NodeCG
-> Logs an empty string

### Root cause
When you assign a string to a replicant, the value stored in *.rep file is not string as per JSON spec.

For example:

```js
const fooRep = nodecg.Replicant('foo')
fooRep.value = "foo"
```
stores
```
foo
```
in foo.rep file, but it should be
```
"foo"
```

And this means an empty foo.rep file means `""` value in the replicant.

**Comment by Hoishin** [comment]
Hmm, actually this only happens when you don't have JSON schema...

**Comment by Lange** [comment]
This is fixed by #521, and will be part of v2.0
